crates/ruff/src/args.rs

```diff
@@ -501,6 +502,9 @@ pub struct ServerCommand {
     /// Enable preview mode; required for regular operation
     #[arg(long)]
     pub(crate) preview: bool,
+    /// Set the number of background threads used for running tasks concurrently - the default is 4.
+    #[arg(long, default_value = "4")]
+    pub(crate) threads: NonZeroUsize,
```

Comment:
This is related to the summary review feedback. The documentation isn't 100% accurate OR requires very specific knowledge of the LSP implementation. The LSP creates one more (background) thread for formatting.  This is what makes me think that it might be best to hide away the thread count from users.

Reply:
Maybe 'worker thread count' would be a better way to describe this. `rust-analyzer` has a [similar setting](https://rust-analyzer.github.io/manual.html#rust-analyzer.numThreads).